techeria_controller/templates/productInfo.html

```diff
@@ -0,0 +1,325 @@
+}
+
+.product_quantity span {
+
+    height: 50px;
+    line-height: 50px;
+    float: left
+}
+
+.product_quantity input {
+    display: block;
+    width: 30px;
+    height: 50px;
+    border: none;
+    outline: none;
+}
+
+.quantity_buttons {
+    position: absolute;
+    top: 0;
+    right: 0;
+    height: 100%;
+    width: 29px;
+}
+
+.quantity_inc,
+.quantity_dec {
+    width: 100%;
+    cursor: pointer
+}
+
+.quantity_control i {
+    font-size: 11px;
+    color: rgba(0, 0, 0, 0.3);
+    pointer-events: none
+}
+
+.quantity_inc {
+    justify-content: flex-end;
+}
+
+</style>
+
+</head>
+
+<body>
+
+  <div class="hero_area">
+    <!-- header section strats -->
+    <header class="header_section">
+      <div class="container-fluid">
+        <nav class="navbar navbar-expand-lg custom_nav-container pt-3">
+          <a class="navbar-brand" href=" {% url 'index' %} ">
+            <span>
+              <img src="/media/images/techeria_logo1.png" style="width: 100px; height: auto;" alt="Techeria">
+            </span>
+          </a>
+          <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+            <span class="navbar-toggler-icon"></span>
+          </button>
+
+          <div class="collapse navbar-collapse" id="navbarSupportedContent">
+            <div class="d-flex ml-auto flex-column flex-lg-row align-items-center">
+              <ul class="navbar-nav">
+                <li class="nav-item active">
+                  <a class="nav-link" href=" {% url 'index' %} ">Home <span class="sr-only">(current)</span></a>
+                </li>
+
+                {% comment %} <li class="nav-item">
+                  <a class="nav-link" href=" {% url 'about' %} "> About </a>
+                </li> {% endcomment %}
+              
+                  <li>
+                  <div class="dropdown">
+                    <button class="dropbtn">Categories</button>
+                    <div class="dropdown-content">
+                      <a href="#">Computers</a>
+                      <a href="#">Phones</a>
+                      <a href="#">Tablets/Ipads</a>
+                      <a href="#">Accessory</a>
+                      <a href="#">Chargers</a>
+                      <a href="#">TV</a>
+                      <a href="#">Speakers</a>
+                      <a href="#">Headsets</a>
+                    </div>
+                  </div>
+                  </li>
+
+                <li class="nav-item">
+                  <a class="nav-link" href=" {% url 'product' %} "> Products </a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link" href=" {% url 'laptop' %} "> Returns/Orders </a>
+                </li>
+
+                {% comment %} <li class="nav-item">
+                  <a class="nav-link" href=" {% url 'contact' %} ">Contact us</a>
```

Comment:
we can opt out from the contact tab .
contact tab is in the index page. let's not carry it over.